var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -23,8 +23,8 @@ class Neuron(Package):
     git      = "https://github.com/nrnhines/nrn.git"
 
     version('develop', branch='master')
+    version('7.6.6',   tag='7.6.6')
```

Comment:
why not mark this as `preferred=True` here?

Reply:
Because then every module that uses `neuron` package will use this new version and there should be further testing in those packages